### ISSUE TITLE
Added features for `Percent_Expressing`

### ISSUE THREAD

**Issue by samuel-marsh** [samuel-marsh/scCustomize#3]
- [ ] Support for setting individual expression thresholds for each feature supplied
- [ ] Support for quantifying cells expressing two or more features simultaneously

**Comment by denvercal1234GitHub** [comment]
Hi @samuel-marsh - This is quite a useful helper function. I was wondering if you could elaborate on how the function determine if a cell is considered positive here? Is it just if it has normalised expression > 0? Thank you!

**Comment by samuel-marsh** [comment]
@denvercal1234GitHub yes with default settings that is correct. This can be modified by altering the threshold parameter or the layer used for calculation.
https://samuel-marsh.github.io/scCustomize/reference/Percent_Expressing.html

Best,
Sam
